server_apps/apache/inc-redirect

```diff
@@ -98,7 +98,7 @@ RewriteCond %{QUERY_STRING}     ^Mival=aa-fxfigureview.apg&OID=([\w-]+)$    [NC]
 RewriteRule ^.*$ /%1? [NC,R=301]
 
 RewriteCond %{QUERY_STRING}     ^Mival=aa-fxallfigures.apg&OID=([\w-]+)$    [NC]
-RewriteRule ^.*$ /action/figure/all-figure-view/%1? [NC,R=301]
+RewriteRule ^.*$ /action/publication/%1/all-figures? [NC,R=301]
 
```

Comment:
should we add a general rewrite rule in case someone follows an old link?

Something like:
/action/figure/all-figure-view/(.*)?(.*) -> /action/publication/$1/all-figures?$2

Reply:
Good idea. However, I'd like to remove the really old redirects from Informix,i.e. Mival=aa- and that sort.